commands/audit/scarunner.go

```diff
@@ -258,7 +261,10 @@ func SetResolutionRepoIfExists(params xrayutils.AuditParams, tech coreutils.Tech
 	if params.DepsRepo() != "" || params.IgnoreConfigFile() {
 		return
 	}
-
+	if tech == coreutils.Pnpm {
```

Comment:
Backward compatibility could be a bit tricky here. We will probably introduce a `jf pnpm` command in the future. Users might execute `jf npmc` before `jf audit` and anticipate its continued functionality with pnpm.

Reply:
I will add pnpm-config command to be used.
https://github.com/jfrog/jfrog-cli/pull/2444